meson.build

```diff
@@ -176,16 +176,8 @@ endif
 
 # tvm
 tvm_dep = dependency('', required: false)
-dlpack_dep = dependency('', required: false)
-if not get_option('tvm-support').disabled()
-  tvm_dep = dependency('tvm', required: false)
-  dlpack_dep = dependency('dlpack', required: false)
-  if not tvm_dep.found()
-    tvm_dep = cxx.find_library('tvm', required: get_option('tvm-support'))
-  endif
-  if not dlpack_dep.found()
-    dlpack_dep = cxx.find_library('dlpack', required: get_option('tvm-support'))
-  endif
+if not get_option('tvm-support').disabled() and cxx.has_header('dlpack/dlpack.h')
```

Comment:
suggestion:
1. remove 177~180 and change args in line 316
```
  'tvm-support': {
    'target': 'tvm_runtime',
    'project_args': { 'ENABLE_TVM' : 1 }
  }
```
2. change tvm_dep to tvm_support_deps in tests dir

Reply:
Looks much better!